satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -2734,3 +2744,260 @@ datasets:
     file_type: nc_fci_asr
     file_key: product_timeliness
     long_name: product_timeliness_index
+
+# AMV Intermediate Product
+  intm_latitude:
+    name: intm_latitude
+    file_type: nc_fci_amvi
+    file_key: intm_latitude
+    standard_name: latitude
+
+  intm_longitude:
+    name: intm_longitude
+    file_type: nc_fci_amvi
+    file_key: intm_longitude
+    standard_name: longitude
+
+  intm_speed:
+    name: intm_speed
+    file_type: nc_fci_amvi
+    file_key: intm_speed
+    standard_name: wind_speed
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_u_component:
+    name: intm_u_component
+    file_type: nc_fci_amvi
+    file_key: intm_u_component
+    standard_name: wind_speed_horizontal_component
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_v_component:
+    name: intm_v_component
+    file_type: nc_fci_amvi
+    file_key: intm_v_component
+    standard_name: wind_speed_vertical_component
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_direction:
+    name: intm_direction
+    file_type: nc_fci_amvi
+    file_key: intm_direction
+    standard_name: wind_to_direction
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_pressure:
+    name: intm_pressure
+    file_type: nc_fci_amvi
+    file_key: intm_pressure
+    standard_name: wind_pressure
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_temperature:
+    name: intm_temperature
+    file_type: nc_fci_amvi
+    file_key: intm_temperature
+    standard_name: wind_temperature
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_target_type:
+    name: intm_target_type
+    file_type: nc_fci_amvi
+    file_key: target_type
+    standard_name: wind_target_type
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_wind_method:
+    name: intm_wind_method
+    file_type: nc_fci_amvi
+    file_key: wind_method
+    standard_name: wind_wind_method
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+# AMV Final Product
+  channel_id:
+    name: channel_id
+    file_type: nc_fci_amv
+    file_key: channel_id
+    standard_name: channel_id
+
+  latitude:
+    name: latitude
+    file_type: nc_fci_amv
+    file_key: latitude
+    standard_name: latitude
+
+  longitude:
+    name: longitude
+    file_type: nc_fci_amv
+    file_key: longitude
+    standard_name: longitude
```

Comment:
According to codefactor, `longitude` and `latitude` are already defined in this yaml file, so I guess we need to rename these?

Reply:
ok I changed them both to amv_latitude/longitude and it seems to work without issue i can still access it via loading lat and lon in the scene and the pytest pass without issues